common/lib/xmodule/xmodule/modulestore/tests/test_mixed_modulestore.py

```diff
@@ -2091,3 +2096,123 @@ def test_import_edit_import(self, default):
                 with self.store.branch_setting(ModuleStoreEnum.Branch.published_only, course_id):
                     published_vertical = self.store.get_item(vertical_loc)
                 self.assertEqual(draft_vertical.display_name, published_vertical.display_name)
+
+    def test_course_publish_signal_firing_split(self):
```

Comment:
Is there a reason these tests are duplicated for split and mongo, rather than using `ddt`?